src/main/webapp/assets/js/userprofile.js

```diff
@@ -0,0 +1,5 @@
+async function getUserInformation() {
+    let response = await fetch('/userinformation');
+    let userinformation = await response.json();
+    return userinformation;
```

Comment:
I suppose this is just a dummy for now right? Will be returning something else or formatting the response gotten as the development goes along?

Reply:
yeap this is a dummy for now. yeap i will modify this along the way.